WHATSNEW.md

```diff
@@ -2,6 +2,12 @@
 
 These are new features and improvements of note in each release.
 
+## v0.3 (TBA)
```

Comment:
Still some problems with this branch not being up-to-date. Maybe redo this PR off of a clean master, or try rebasing.